dnf5daemon-server/main.cpp

```diff
@@ -39,6 +39,8 @@ int main() {
     sigaddset(&mask, SIGINT);
     sigaddset(&mask, SIGTERM);
     pthread_sigmask(SIG_BLOCK, &mask, NULL);
+    // ignore SIGPIPE globally, handle pipe writing errors locally where they occur
```

Comment:
Do you mean `locally when they occur`?

Reply:
Actually I meant `where`. The pipe errors are handled in `write_to_fd()` function during attempt to write.
Without blocking, the `SIGPIPE` signal is emitted instead of returning an errno and the program stops.

Reply:
Thank you